services/auth-server/app/app/__init__.py

```diff
@@ -15,9 +19,14 @@
 
 
 def create_app(config_class=None):
-    app = Flask(__name__)
+    app = Flask("auth-server")
```

Comment:
Seems like the app name has some implications related to templates and static files. In this particular case, the `auth-server` will not be able to find templates.

Not sure if we want to simply add the `template_folder` arg or actually rename the `app` directory since the app name has also other implications, [docs](https://flask.palletsprojects.com/en/1.1.x/api/#application-object) for reference.

```suggestion
    app = Flask("auth-server", template_folder="app/templates")
```

Reply:
I will look into the suggestions, right now I would say I really like the `template_folder` arg 👍